app.rb

```diff
@@ -0,0 +1,120 @@
+require_relative 'person'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'classroom'
+require_relative 'book'
+require_relative 'rental'
+require 'pry'
+
+class App
+  attr_reader :books, :person, :rentals, :classroom
+
+  def initialize
+    @books = []
+    @person = []
+    @rentals = []
+    # @classroom = Classroom.new('college')
+  end
+
+  # Method to list all books
+  def list_all_books
+    puts 'All Books:'
+    @books.each do |book|
+      puts "Title: #{book.title}, Author: #{book.author}"
+    end
+  end
+
+  # Method to list all people
+  def list_all_people
+    puts 'All People:'
+    @person.each do |person|
+      if person.instance_of?(Student)
+        puts "[Student]: ID: #{person.id}, Name: #{person.name}, age: #{person.age}"
+      else
+        puts "[Teacher]: ID: #{person.id}, Name: #{person.name}, age: #{person.age}"
+      end
+    end
+  end
```

Comment:
- [ ] [optional] @Jeddaa It would be nice if you provide a message when there are no users added to the app instead of it displaying an empty space, it also applies to the `list_all_books` method
![image](https://github.com/Jeddaa/school-library-oop/assets/61408860/6eca9749-c345-4c57-b2a6-ebab14db538e)